scripts/init/vyos-router

```diff
@@ -173,6 +173,15 @@ bind_mount_boot ()
 
 start ()
 {
+    log_daemon_msg "Waiting for NIC's to settle down"
+    # On boot time udev migth take a long time to reorder nic's, this will ensure that
+    # all udev activity is completed and all nics presented at boot-time will have their
+    # final name before continuing with vyos-router initialization.
+    SECONDS=0
+    udevadm settle
+    STATUS=$?
+    log_progress_msg "setteled in ${SECONDS}sec."
```

Comment:
where is the new value set for SECONDS?

Reply:
This is an obscure bash built-in, an auto-incrementing timer.